### PR TITLE
Minor code cleanup

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Core/src/Result.php [hunk 1]
```diff
@@ -24,7 +25,7 @@ class Result
      */
     protected $input;
 
-    protected $initialized = false;
+    private $initialized = false;
```
Comment (Nyholm) [inline src/Core/src/Result.php:28]: This is currently only used in the `Result` class.

File: src/Core/src/Waiter.php [hunk 2]
```diff
@@ -48,13 +48,6 @@ class Waiter
      */
     private $finalState;
 
-    /**
-     * A Result can be resolved many times. This variable contains the last resolve result.
-     *
-     * @var bool|NetworkException|null
-     */
-    private $resolveResult;
```
Comment (Nyholm) [inline src/Core/src/Waiter.php:56]: This was never used. I guess we missed removing it when introducing the `Response` class

**Comment by jderusse** [comment]
thanks

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**